src/App.jsx

```diff
@@ -1,12 +1,30 @@
 import { useState } from 'react'
 import './App.css'
+import { Routes, Route, Router } from 'react-router-dom'
+import { Home } from './components/Home'
+import { Explore } from './components/Explore'
+import { Post } from './components/Post'
+import { LogIn } from './components/LogIn'
+import { Account } from './components/Account'
+import { Header } from './components/Header'
 
 function App() {
 
+  const [articles, setArticles] = useState([])
+
   return (
-    <>
-    
-    </>
+  <>
+      <Header></Header>
```

Comment:
You don't need a closing tag for rendering Components unless you were going to use it's children.